documentation/staging/content/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -16,14 +16,21 @@ WebLogic Server supports two types of clustering configurations, configured and
 
 For more in-depth information on support for scaling WebLogic clusters in Kubernetes, see [WebLogic Dynamic Clusters on Kubernetes](https://blogs.oracle.com/weblogicserver/weblogic-dynamic-clusters-on-kubernetes).
 
+WebLogic Kubernetes Operator 4.0 introduces a new Kubernetes resource, called a Cluster resource, that is of custom resource type `Cluster`.  Cluster resources are monitored by the WebLogic Kubernetes operator. A cluster resource references an individual WebLogic cluster and its configuration.  It also controls how many member servers are running along with potentially any additional Kubernetes resources that are specific to that WebLogic cluster.  The Cluster resource enables the `/scale` subresource which allows it to be scaled up or down via an HTTP endpoint.
```

Comment:
You can simplify these first two sentences to, "WebLogic Kubernetes Operator 4.0 introduces a new custom resource, `Cluster`.

Reply:
Oh, we also need to highlight the support for autoscaling here in the overview.